docs/reference/modules/http.md

```diff
@@ -105,10 +106,10 @@ Accepts the following properties on the optional `options` object:
 
 | Option | Type |Description |
 | -------- | ------ | ----------- |
-| `qs` | Object | An object of query parameters and values. |
+| `qs` | Object | An object of query parameters and values. It may contain nested objects, arrays, and properties with null values. |
 | `jar` | Object | Pass in a cookie jar obtained from `apos.http.jar()`. |
 | `send` | String | Can be 'json' to always send `options.body` JSON-encoded, or `form` to send it URL-encoded. |
-| `body` | Variable | The request body. If an object or array, it is sent as JSON. Otherwise it is sent as-is, unless the `send` option is set. |
+| `body` | Variable | The request body. If an object or array, it is sent as JSON. If a `FormData` object, it is sent as a `multipart/form-data` upload. Otherwise it is sent as-is, unless the `send` option is set. |
```

Comment:
`FormData` doesn't just exist in server land, so this should link specifically to:

https://www.npmjs.com/package/form-data

Which is known to be compatible.